Configuration/StandardSequences/python/DD4hep_GeometrySim_cff.py

```diff
@@ -1,3 +1,3 @@
 import FWCore.ParameterSet.Config as cms
 
-from Configuration.Geometry.GeometryDD4hepExtended2021_cff import *
```

Comment:
@slomeo are you changing this on purpose? We should be a bit careful with changing the StandardSequences.
In this case, it affects only test code, so it should not be a big problem.
In any case, @civanch @cvuosalo  are you ok with this change? 
```
Geometry/CSCGeometryBuilder/test/python/validateCSCGeometryDD4hep_cfg.py  
Geometry/GEMGeometryBuilder/test/python/validateGEMGeometry_cfg.py 
Geometry/RPCGeometryBuilder/test/python/validateRPCGeometry_cfg.py 	
Geometry/TrackerNumberingBuilder/test/trackerModuleNumberingDD4hep_cfg.py
SimG4Core/DD4hepGeometry/test/python/testZMM_13TeV_TuneCUETP8M1_cfi_GEN_SIM.py
```

Reply:
@silviodonato : I changed it only for testing if the DD4HEP Geometry was the same of DDD (and Reco). I had to do it because ME0 is present only in 2026Scenario (an not in 2021)

Reply:
@slomeo - IMHO you should add a separate sequence for a 2026 scenario, and not modify the 2021 one which is heading for production

Reply:
@ianna : do you mean that it's better to create DD4hep_GeometrySimBis_cff.py (note Bis) in which I put GeometryDD4hepExtended2026D41_cff?

Reply:
I agree with @ianna 
It could be called DD4hep_GeometrySimPhase2_cff.py (not sure what "Bis" means)

Reply:
@silviodonato @ianna @kpedro88 : I restored Configuration/StandardSequences/python/DD4hep_GeometrySim_cff.py and I created restored Configuration/StandardSequences/python/DD4hep_GeometrySimPhase2_cff.py